sdmetrics/reports/multi_table/diagnostic_report.py

```diff
@@ -33,6 +33,10 @@ class DiagnosticReport():
         'Boundaries': [BoundaryAdherence],
     }
 
+    _METRIC_ARGS = {
```

Comment:
Is there a reason to make this a class attribute instead of an instance attribute? I just wonder because if you change the values in one then I think the next `DiagnosticReport` created will also have the changed value

Reply:
Ah I see, I can move it to an instance attribute, good call